regolith/builders/preslistbuilder.py

```diff
@@ -1,5 +1,24 @@
-"""Builder for Lists of Presentations."""
-from copy import deepcopy
+"""Builder for Lists of Presentations.
+
+This builder will build a presentation list for each group-member in each group
+listed in groups.yml.  Group members are indicated in the employment and
+education sections of the individual in people.yml.
+
+There are a number of filtering options, i.e., for presentation-type (invited,
+colloquium, seminar, poster, contributed_oral) and for whether the invitation
+was accepted or declined.  As of now, these filters can only be updated by
+editing this file but may appear as command-line options later.  It will also
```

Comment:
Do you want me to put up a PR adding these options?

Reply:
@CJ-Wright do you mean command-line options?

Definitely things are heading in that direction for me anyway.  I sometimes need all my invites (for the Dean) but sometimes only the talks I gave (for my cv or my grant report) and the Dean wants it broken out by seminar vs invited vs plenary and the grant just wants invited talks, and so on and so on and so on.  I don't think it is a good idea to build a new builder for every option, so capturing the most common ones with filters and allowing to specify which filters at runtime seems like a good way to go.

Reply:
Yes, we could add some optional kwargs to the CLI which could specify these different situations. That would prevent us from having to build a ton of builders (which would be painful).